### PR TITLE
Updated docs to use Ember.Controller

### PR DESCRIPTION
**Pull request by Patsy-issa**
ObjectController is deprecated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ seen as a validatable object.
 import Ember from 'ember';
 import EmberValidations from 'ember-validations';
 
-export default Ember.ObjectController.extend(EmberValidations, {
+export default Ember.Controller.extend(EmberValidations, {
   validations: {
     firstName: {
```
Comment (bcardarella) [inline README.md]: with this change this will have to move over to `model.firstName`

Comment (Patsy-issa) [inline README.md]: Ah right, missed it, I ll update it now.

**Comment by Patsy-issa** [comment]
Updated to use `model.property`

File: README.md [hunk 2]
```diff
@@ -64,16 +64,16 @@ seen as a validatable object.
 import Ember from 'ember';
 import EmberValidations from 'ember-validations';
 
-export default Ember.ObjectController.extend(EmberValidations, {
+export default Ember.Controller.extend(EmberValidations, {
   validations: {
-    firstName: {
+    model.firstName: {
```
Comment (bcardarella) [inline README.md]: it has to be in quotes `"mode.firstName"`

**Comment by Patsy-issa** [comment]
Wrapped the object keys in quotes.

File: README.md [hunk 3]
```diff
@@ -87,7 +87,7 @@ import EmberValidations from 'ember-validations';
 
 export default Ember.Component.extend(EmberValidations, {
   validations: {
-    'user.firstName': {
+    'model.user.firstName': {
```
Comment (bcardarella) [inline README.md]: This one doesn't need to be `model.user.firstName` as it looks like the intent here is to operate from the `user` attribute so `user.firstName` works

Comment (Patsy-issa) [inline README.md]: :+1:

**Comment by Patsy-issa** [comment]
Updated.

**Comment by bcardarella** [comment]
Thanks!

**Pull request merged**